src/tools/devtools/extensions.md

```diff
@@ -0,0 +1,41 @@
+---
+title: DevTools extensions
+description: Learn how to use and build DevTools extensions.
+---
+
+## What are DevTools extensions?
+
+[DevTools extensions](https://pub.dev/packages/devtools_extensions) are developer
+tools provided by third-party packages that are tightly integrated into the
+DevTools tooling suite. Extensions are distributed as part of a pub package,
+and they are dynamically loaded into DevTools when a user is debugging their app.
+
+## Using a DevTools extension
+
+If your app depends on a package that provides a DevTools extension, the
+extension will automatically show up in a new tab when you open DevTools. You
+will need to manually enable the extension before it loads for the first time.
+Make sure the extension is provided by a source you trust before enabling it.
+
+![Screenshot of extension enablement prompt]({{site.url}}/assets/images/docs/tools/devtools/extension_enable_prompt.png)
+
+### devtools_options.yaml
```

Comment:
What about "Extension enablement states"? This may not always be an extensions only file and the user shouldn't need to manually configure this. This doc is supposed to be an explanation of why a `devtools_options.yaml` file is showing up in the user's project and what they should do with it.

Reply:
Is "Configure extension enablement states" ok? I'm trying to keep the headers parallel,  in the sense that we try to start each header with a (non-gerund) verb. But if it's really not working for you, I can be flexible.

Reply:
I used "Configure extension enablement states" and moved the screenshot and explanation of how to enable the extension from devtools to this section. I think this makes it more clear that you should be enabling and disabling extensions from DevTools, and that the file is just to persist this data.